<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>No. of data points of X or Y,n =</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>0</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <number>2</number>
   </property>
   <property name="maximum">
    <number>6</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>X Values</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Y Values</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>391</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>The coefficients of X for the found quadratic polynomial:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>80</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>a0=</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>120</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>a1=</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>160</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>a2=</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_1">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>70</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>110</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_3">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>150</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>241</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>The Quadratic Polynomial is:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="START">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>200</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>FIND</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Y(X)=</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>280</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>290</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_5">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>280</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>290</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>290</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_6">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>280</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>290</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>X^2</string>
   </property>
  </widget>
  <widget class="QCustomPlot" name="customPlot" native="true">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>350</y>
     <width>321</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>67</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>GRAPH:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="secwindow">
   <property name="geometry">
    <rect>
     <x>458</x>
     <y>480</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Inputs For Test Run</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>117</width>
     <height>194</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_7">
      <property name="decimals">
       <number>8</number>
      </property>
      <property name="minimum">
       <double>-5000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>5000.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.000001000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_8">
      <property name="decimals">
       <number>8</number>
      </property>
      <property name="minimum">
       <double>-5000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>5000.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.000001000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_9">
      <property name="decimals">
       <number>8</number>
      </property>
      <property name="minimum">
       <double>-5000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>5000.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.000001000000000</double>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_10">
      <property name="decimals">
       <number>8</number>
      </property>
      <property name="minimum">
       <double>-5000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>5000.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.000001000000000</double>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_11">
      <property name="decimals">
       <number>8</number>
      </property>
      <property name="minimum">
       <double>-5000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>5000.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.000001000000000</double>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_12">
      <property name="decimals">
       <number>8</number>
      </property>
      <property name="minimum">
       <double>-5000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>5000.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.000001000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>121</width>
     <height>194</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_1">
      <property name="decimals">
       <number>8</number>
      </property>
      <property name="minimum">
       <double>-5000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>5000.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.000001000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
      <property name="decimals">
       <number>8</number>
      </property>
      <property name="minimum">
       <double>-5000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>50000.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.000001000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
      <property name="decimals">
       <number>8</number>
      </property>
      <property name="minimum">
       <double>-5000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>5000.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.000001000000000</double>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_4">
      <property name="decimals">
       <number>8</number>
      </property>
      <property name="minimum">
       <double>-5000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>5000.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.000001000000000</double>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_5">
      <property name="decimals">
       <number>8</number>
      </property>
      <property name="minimum">
       <double>-5000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>5000.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.000001000000000</double>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_6">
      <property name="decimals">
       <number>8</number>
      </property>
      <property name="minimum">
       <double>-5000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>5000.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.000001000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_RESET">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>200</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>RESET</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Close2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>CLOSE</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
